doc/rados/operations/control.rst

```diff
@@ -100,7 +100,7 @@ Write a copy of the most recent OSD map to a file. See
 
 	ceph osd getmap -o file
 
-.. _osdmaptool: ../../man/8/osdmaptool
+.. _osdmaptool: http://docs.ceph.com/docs/master/man/8/osdmaptool/
```

Comment:
it should be
```rst
.. _osdmaptool: /man/8/osdmaptool
```

or better off using the `:ref:` directive as @alfredodeza suggests.

Reply:
@tchaikov I added a ``reST`` label for ``osdmaptool`` (I didn't see a reference label above the section title). 

This should work. If I missed something, please let me know.